Components/navigation.css

```diff
@@ -0,0 +1,54 @@
+
+.navigation{
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+    width:80%;
+    height:40px;
+    font-size: 25px;
+    color: var(--light-color);
+    background-color: var(--button-floatHover-color);
+    border-radius: 5px;
+    padding:20px;
+}
+.left-of-nav{
+    cursor: pointer;
+    display: flex;
+    width: 35%;
+    justify-content: space-between;
+    align-items: center;
+}
+.right-of-nav{
+    display: flex;
+    width:40%;
+    justify-content: space-evenly;
+    align-items: center;
+}
+.nav-logo{
+    font-family: 'Dancing Script';
+    font-size:30px;
+    text-shadow:2px 2px 5px var(--primary-color);
```

Comment:
Suggestion: Text Shadow can be removed to make it more generic. If required somewhere can be added then explicitly.